tests/unit/amo/components/TestHeadLinks.js

```diff
@@ -23,11 +23,16 @@ describe(__filename, () => {
     'renders alternate links with hreflang for %s',
     (clientApp) => {
       const baseURL = 'https://example.org';
+      const lang = 'en-US';
       const to = '/some-url';
 
       const _hrefLangs = ['fr', 'en-US'];
       const _config = getFakeConfig({ baseURL });
-      const { store } = dispatchClientMetadata({ clientApp });
+      const { store } = dispatchClientMetadata({
+        clientApp,
+        lang,
```

Comment:
Nit: I was trying to follow why these changes were needed to the test, so I commented out this line and found that the test still passed. I realized that this is because `en-US` is the default value for `lang` in `dispatchClientMetadata`, so maybe it would be better to set `lang` to a different value than `en-US`.